tests/upgrades/test_yum_plugins.py

```diff
@@ -112,10 +111,8 @@ def _install_or_update_package(self, client_container_id, package, update=False)
 
     def _create_custom_rhel_tools_repos(self, product):
         """Install packge on docker content host."""
-        rhel_repo_url = os.environ.get('{}_CUSTOM_REPO'.format(self.client_os.upper()))
-
-        tools_repo_url = os.environ.get(
-            'TOOLS_{}'.format(self.client_os.upper()))
+        rhel_repo_url = settings.rhel7_os
+        tools_repo_url = settings.sattools_repo[DISTRO_RHEL7]
```

Comment:
here also we hardcoding to rhel7, earlier it was dynamically generated anything (rhel6,7,8). we need to have some dynamic nature or adding if else would make sense.

Reply:
Same as above.